pset-iterators.js

```diff
@@ -114,7 +123,16 @@ Expected Result Shape: An array of objects (with each object representing a pres
 console.log('Problem 4:')
 
 // Add your code below this line
-
+const presidentsBetween1850and1900 = presidents.filter((president) => {
+  const tookOfficeYear = parseInt(president.took_office.split("-")[0])
+  if (tookOfficeYear > 1850 && tookOfficeYear < 1900){
+    return true
+  } else {
+    return false
+  }
+})
+
+console.log(presidentsBetween1850and1900)
```

Comment:
@jl1982 Good Job here using this variable `tookOfficeYear` to retrieve the year value from the `took_office` value. Then using the conditional to check if that year falls between 1850 and 1900.

Take a look at this alternate approach [here](https://github.com/jsr20190826/hw-03-iterators-solution/blob/96bab20deba3ad116374a5827291a951faf5d555/pset-iterators.js#L120)